### DISCUSSION TITLE
RN finds items, but does not download them.

### DISCUSSION THREAD

**Discussion by Davikar** [trickerer01/Ruxx#8] [Q&A] [answered]
So, I'm not sure if I am using this wrong or something.
I have filled out the required session cookies for the RN module (shm_user, shm_session and cf_clearance).
I have set videos to prefer MP4 and images to prefer high res.
Hitting the check button reveals there's 39 items.
And then when hitting the download button it processes the items, but doesn't actually download any of them:

```
Total 39 items found across 1 pages
Looking for min page by date...
Only a page or two! Skipping
Looking for max page by date...
Only a page or two! Skipping
new totalcount: 39
Scanning pages 1 - 1
page: 1 / 1	(1-39 / 39)
new totalcount: 39
Filtering trailing back items...
last items filter is irrelevant! Skipping
new totalcount: 39
Filtering trailing front items...
first items filter is irrelevant! Skipping
new totalcount: 39
Filtering items by type...
Both types are enabled! Skipping
new totalcount: 39
Filtering items by previous tasks...
new totalcount: 39
Filtering out existing items...
new totalcount: 39
Filtering out items using custom filters...
new totalcount: 39

Processing 39 items, bound ###### to ######
39 item(s) scheduled, 4 thread(s) max
Working...


All processed (39 items)...

[21:36:30] 1 task(s) completed, 0 / 0 item(s) succeded
```

**Comment by trickerer01** [comment, marked as answer]
Confirmed. This a bug actually. Seems like when using RN credentials the 'post comment' section is considered a valid comment but attempt to extract it is doomed to fail. Better make sure stacktrace is printed when something like this happens, huh.
Thankfully credentials aren't always needed.
The fix in underway.

**Reply by trickerer01** [reply]
https://github.com/trickerer01/Ruxx/releases/tag/2023.09.20
